config.js

```diff
@@ -7,12 +7,15 @@ module.exports = {
     url: process.env.SQS_URL
   },
   db: {
-    database: process.env.DATABASE_NAME || 'taskflow',
+    database: process.env.DATABASE_NAME || 'asl',
     host: process.env.DATABASE_HOST,
     password: process.env.DATABASE_PASSWORD,
     port: process.env.DATABASE_PORT,
     user: process.env.DATABASE_USERNAME || 'postgres'
   },
+  taskflowdb: {
```

Comment:
not camel case?

Reply:
taskFlowDb, taskFlow DB, taskflowDb, taskflowDB